src/screens/Auth/RegisterStack/CreatePasswordScreen/CreatePasswordScreen.tsx

```diff
@@ -35,11 +37,9 @@ export default function CreatePasswordScreen(props: Props) {
         'Content-Type': 'application/json',
       },
       body: JSON.stringify({
-        'Full Name': user.name,
         'Email': user.email,
-        'Current Title': user.userTitle,
-        'Company Code': user.companyCode,
         'Password': password,
+        '_record': user.userRecord,
```

Comment:
Can you remove the if(data.ID = undefined) statement on line 48. It won't ever be used. 
(Had to comment here because you didn't edit those lines so github wont allow me to)